recipes/motmetrics/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "motmetrics" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7328d8468c948400b38fcc212f3e448bc1f2fdfc727e170d85a029e49f1cdbc6
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - enum34
```

Comment:
If we're not worried about running on Python < 3 any more, do we need this at all?

Reply:
probably not :wink: